GestaoFinanceira/GestaoFinanceira/BD/DAO/CreditCardDAO.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GestaoFinanceira.BD.Conections;
+using GestaoFinanceira.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GestaoFinanceira.BD.DAO
+{
+    public class CreditCardDAO : DAOBase<CreditCard>
+    {
+
+        public CreditCardDAO(IConnection<CreditCard> connection) : base(connection)
+        {
+
+        }
+
+        public override bool Delete(int id)
+        {
+            return connection.Delete(id);
+        }
+
+        public override IEnumerable<CreditCard> Get()
+        {
+            return connection.Select();
+        }
+
+        public override CreditCard Insert(CreditCard creditCard)
+        {
+            var id = connection.Insert(creditCard);
+            creditCard.Id = id;
+            return creditCard;
+        }
+
+        public override CreditCard Update(CreditCard creditCard)
+        {
+            connection.Update(creditCard);
+            return creditCard;
+        }
+
+        protected override string GetTableName()
+        {
+            return "CreditCard";

```

Comment:
Pra banco de dados, é normal utilizar outro padrão, tudo em caixa baixa e separado com `-` exemplo:

```suggestion
            return "credit-card";

```

Reply:
Gostei da ideia, acho que vale a pena criar um cartão para essas edições também, ou se quiser aplicar fique a vontade.